src/main/java/org/apidesign/polyfill/Polyfill.java

```diff
@@ -0,0 +1,8 @@
+package org.apidesign.polyfill;
+
+import org.graalvm.polyglot.Context;
+
+public interface Polyfill {
+
+    void initialize(Context ctx);
```

Comment:
Isn't this over designed? We pretend there are independent _polyfill_ components, but they are not that much independent.